### PR TITLE
Parents page

### PR DESCRIPTION
**Pull request by xtine**
also more accessibility updates and various design bug fixes

### CODE REVIEW

**Comment by OpenGlobe** [comment]
Just waiting on NPS research link / info...

**Comment by OpenGlobe** [comment]
@khandelwal - we're good to go.

**Comment by xtine** [comment]
@khandelwal now also has 404 page.

File: ekip/everykid/urls.py [hunk 1]
```diff
@@ -52,7 +52,16 @@
...
+    # PARENTS
+    url(r'parents/$', cache_page(60*60)(TemplateView.as_view(
+        template_name="parents.html")), name="parents"),
+
     # LEGAL
     url(r'privacy-policy/$', cache_page(60*60)(TemplateView.as_view(
         template_name="legal/privacy.html")), name="privacy_policy"),
+
+    # 404
+    url(r'not-found/$', cache_page(60*60)(TemplateView.as_view(
```
Comment (khandelwal) [inline ekip/everykid/urls.py]: We should probably remove this URL.

**Comment by khandelwal** [comment]
That coveralls build has been pending for a while. I'm just going to merge.

**Pull request merged**